.github/workflows/release-drafter.yml

```diff
@@ -9,7 +9,27 @@ jobs:
   update_release_draft:
     runs-on: ubuntu-latest
     steps:
-      - name: Draft release notes
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          # Only need package.json as it is used by the get_version step to pass tag number into release-drafter
+          sparse-checkout: package.json
+
+      - name: Get version number from package.json
+        id: get_version
+        shell: bash
+        run: |
+          set -e
+          echo "version_number=$(jq -r '.version' package.json)" >> $GITHUB_OUTPUT
+
+      - name: Draft release notes for v${{ steps.get_version.outputs.version_number }}
         uses: release-drafter/release-drafter@v6
+        with:
+          # Setting tag and name override the name-template and tag-template.
+          # See https://github.com/release-drafter/release-drafter?tab=readme-ov-file#action-inputs for more
+         tag: "v${{ steps.get_version.outputs.version_number }}"
+         name: "v${{ steps.get_version.outputs.version_number }}"
+         version: "v${{ steps.get_version.outputs.version_number }}"
```

Comment:
Not sure if it matters, but is version expected to be a version number (just `1.0.0`), or can it be anything (with `v` in front)?

Reply:
@bartaz `tag` is filled into the tag field for the release. It is subject to some validation from Github (it has to be a [git ref](https://git-scm.com/docs/git-check-ref-format)). 

`name` is filled into the name of the release. 

`version` is used for the body of the release (where it says "New in Vanilla v{x}"). 

Here's an example image where each field is different:

![Screenshot 2024-05-08 at 1 39 52 PM](https://github.com/canonical/vanilla-framework/assets/46915153/792d9257-f692-481d-b219-88b4cfc55653)

Reply:
@jmuzina Thanks. So in this case `v` would be duplicated in the body? Because we add it both here into the `version` param, and then again in the template as `v$INPUT_VERSION`. That's why I guess we may want to put just the number of the version into version param. I also makes more sense semantically.

```suggestion
         version: "${{ steps.get_version.outputs.version_number }}"
```

Reply:
I'm glad you brought this up. It seems like release drafter is doing some sort of logic against `input.tag` to split off the `v` so that `$INPUT_VERSION` template has no `v`. Here's an example of that where you can see that inputting `v4.12.1` as `tag` causes the tag to be resolved to `v4.12.1` and `$INPUT_VERSION` as `4.12.1`.

So, we don't need `input.version` at all in this case; we can just specify `name` for the draft release name and `tag` for the tag to use :)

![Screenshot 2024-05-08 at 3 03 30 PM](https://github.com/canonical/vanilla-framework/assets/46915153/41f84415-ed6c-44ec-af14-df8c8f3e6400)